subprojects/gradle/runner/report/src/main/kotlin/com/avito/android/runner/report/Report.kt

```diff
@@ -1,27 +1,10 @@
 package com.avito.android.runner.report
 
-import com.avito.android.Result
 import com.avito.report.model.AndroidTest
-import com.avito.report.model.CrossDeviceSuite
-import com.avito.report.model.TestStaticData
 
-public interface Report : ReadReport {
+public interface Report : ReadReport, LegacyReport {
```

Comment:
Can we make the opposite direction of inheritance?

`interface LegacyReport: Report`

If I need an acces to legacy API it'd better to make it explicitly.

Reply:
i separated it completely now